src/components/GenreSelector/GenreSelector.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const GenreSelector = ({ genres, selectedGenre, onSelect }) => {
+  return (
+    <div>
+      {genres.map(genre => (
```

Comment:
You can use an `index` for `key` prop, so if you have duplicate of genre, it wont have an issue:

```jsx
  {genres.map((genre, index) => (
    <button
      key={index}
```